frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -3,25 +3,28 @@ import type { FC } from 'react'
 import { useDBStructureStore } from '../../../stores'
 import { convertDBStructureToNodes } from '../convertDBStructureToNodes'
 import styles from './ERDContent.module.css'
+import { RelationshipEdge } from './RelationshipEdge'
 import { TableNode } from './TableNode'
 
 const nodeTypes = {
   table: TableNode,
 }
 
+const edgeTypes = {
+  relationship: RelationshipEdge,
+}
+
 export const ERDContent: FC = () => {
   const dbStructure = useDBStructureStore()
-  const nodes = convertDBStructureToNodes(dbStructure)
+  const { nodes, edges } = convertDBStructureToNodes(dbStructure)
 
   return (
     <div className={styles.wrapper}>
       <ReactFlow
         nodes={nodes}
+        edges={edges}
         nodeTypes={nodeTypes}
-        defaultEdgeOptions={{
-          /* NOTE: Ensure that the Edge always appears above the Node (Note: The z-index of a selected Node is 1000). */
-          zIndex: 1001,
-        }}
```

Comment:
I was hesitant to delete this code, but since it seemed unnecessary, I removed it. (I have a feeling it might be needed in the future, though💭.)

Reply:
@FunamaYukina 
This setting is necessary for group nodes with parent-child relationships, so it's okay to remove it for now!

Reply:
I see. Thank you for your support👍✨